From a3b6c890b0b3fa9a1362dd331c2fece45cd272ef Mon Sep 17 00:00:00 2001
From: Monika Alekhya <malekh@codeaurora.org>
Date: Tue, 9 Apr 2013 15:57:17 +0530
Subject: [PATCH 165/176] Camera: Fix crash when videoHDR is on with VGA
 resolution.

set low power mode when videoHDR is enabled.

Change-Id: I750385b013f0554e394ecaf60ef840bbb183f5a2
CRs-Fixed: 471607
---
 QCamera/HAL/core/src/QCameraHWI_Parm.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/QCamera/HAL/core/src/QCameraHWI_Parm.cpp b/QCamera/HAL/core/src/QCameraHWI_Parm.cpp
index 4deb4a7..7882e5b 100755
--- a/QCamera/HAL/core/src/QCameraHWI_Parm.cpp
+++ b/QCamera/HAL/core/src/QCameraHWI_Parm.cpp
@@ -2605,7 +2605,7 @@ status_t QCameraHardwareInterface::setPowerMode(const QCameraParameters& params)
     if (powermode != NULL) {
         value = attr_lookup(power_modes,
                 sizeof(power_modes) / sizeof(str_map), powermode);
-        if((value == LOW_POWER) || mHFRLevel > 1) {
+        if((value == LOW_POWER) || mHFRLevel > 1 || (mVideoHDRMode == true)) {
             ALOGI("Enable Low Power Mode");
             value = LOW_POWER;
             mPowerMode = value;
-- 
1.8.3.1

